### PR TITLE
JAMES-2367 Fix JPA flacky test

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: server/container/guice/jpa-guice/src/test/java/org/apache/james/JPAJamesServerTest.java [hunk 1]
```diff
@@ -80,14 +80,18 @@ public void jpaGuiceServerShouldUpdateQuota() throws Exception {
...
+            .hasAMessage());
 
         assertThat(
             imapMessageReader.connect(LOCALHOST, IMAP_PORT)
                 .login(USER, PASSWORD)
                 .getQuotaRoot(IMAPMessageReader.INBOX))
             .isEqualTo("* QUOTAROOT \"INBOX\" #private&toto@james.local\r\n" +
                 "* QUOTA #private&toto@james.local (STORAGE 12 50)\r\n" +
-                "AAAB OK GETQUOTAROOT completed.\r\n");
+                "AAAE OK GETQUOTAROOT completed.\r\n");
```
Comment (mbaechler) [inline server/container/guice/jpa-guice/src/test/java/org/apache/james/JPAJamesServerTest.java]: how do you know what the tag should be ?

Comment (rouazana) [reply server/container/guice/jpa-guice/src/test/java/org/apache/james/JPAJamesServerTest.java]: prescience. Or maybe that fact that there are 5 imap operations done.
but maybe I can replace isEqualTo be startsWith

Comment (mbaechler) [reply server/container/guice/jpa-guice/src/test/java/org/apache/james/JPAJamesServerTest.java]: yes we can

**Comment by rouazana** [comment]
```
[c1b0069aa765100d6ccc137d6928bef38e41e6ed] Failed tests: 
[c1b0069aa765100d6ccc137d6928bef38e41e6ed]   CassandraMailboxRecentDAOTest.getRecentMessageUidsInMailboxShouldNotTimeoutWhenOverPagingLimit:130 
[c1b0069aa765100d6ccc137d6928bef38e41e6ed] Expected size:<6000> but was:<7000> in:
```
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**